elasticsearch/src/main/java/org/hibernate/search/elasticsearch/schema/impl/DefaultElasticsearchSchemaTranslator.java

```diff
@@ -214,17 +213,17 @@ else if ( SpatialHelper.isSpatialFieldLatitude( propertyPath ) ) {
 		}
 	}
 
-	private void addPropertyMapping(ElasticsearchMappingBuilder mappingBuilder, FacetMetadata facetMetadata) {
-		String propertyPath = facetMetadata.getAbsoluteName();
+	private void addFieldMapping(PropertyMapping propertyMapping, ElasticsearchMappingBuilder mappingBuilder, FacetMetadata facetMetadata) {
```

Comment:
Yes, that's really annoying: what we call "fields" in Lucene are called "properties" in ES, and ES gives "fields" a totally different meaning.

I renamed to `addSubfieldMetadata`. Not very correct, but at least it's not ambiguous.